{{cookiecutter.project_shortname}}/site/{{cookiecutter.project_package}}/config.py

```diff
@@ -0,0 +1,7 @@
+"""Custom code config."""
```

Comment:
I find it confusing to have the config both here and in invenio.cfg.....what if they change the variable in invenio.cfg, then they have to copy paste from here.